### PR TITLE
update deprecation notice

### PR DESCRIPTION
**Pull request by Dan33l**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,13 +59,21 @@ include snmp
...
 
-The parameters `ro_community`, `rw_community`, `ro_network`, and `rw_network` will be removed in version 4.0.0 of this module.  The snmptrapd parameter name will become `authcommunity`.
+##### Current module 4.x serie
 
-Support for Puppet 2.7 will be removed in version 4.0.0 of this module.
+ * The parameter `install_client` is renamed to `manage_client`.
+
+ * Support for Puppet lower than version 4 is removed.
+
+##### Futur module 5.x serie
```
Comment (ekohl) [inline README.md]: future?

File: README.md [hunk 2]
```diff
@@ -59,13 +59,21 @@ include snmp
 
 #### Deprecation Warning
 
-The classes `snmp::server` and `snmp::trapd` will be merged into class `snmp` in version 3.0.0 of this module.  All of their class parameters will be made available in the `snmp` class.
+##### Past module 3.x serie
```
Comment (ekohl) [inline README.md]: Not sure, but I think this should be series

File: README.md [hunk 3]
```diff
@@ -59,13 +59,21 @@ include snmp
...
-The parameter `install_client` has been renamed to `manage_client` in version 4.0.0 of this module.
+ * The classes `snmp::server` and `snmp::trapd` has been merged into class `snmp`.  All of their class parameters available in the `snmp` class.
 
-The parameters `ro_community`, `rw_community`, `ro_network`, and `rw_network` will be removed in version 4.0.0 of this module.  The snmptrapd parameter name will become `authcommunity`.
+##### Current module 4.x serie
 
-Support for Puppet 2.7 will be removed in version 4.0.0 of this module.
+ * The parameter `install_client` is renamed to `manage_client`.
+
+ * Support for Puppet lower than version 4 is removed.
```
Comment (ekohl) [inline README.md]: lower -> older?

File: README.md [hunk 4]
```diff
@@ -59,13 +59,21 @@ include snmp
 
 #### Deprecation Warning
 
-The classes `snmp::server` and `snmp::trapd` will be merged into class `snmp` in version 3.0.0 of this module.  All of their class parameters will be made available in the `snmp` class.
+##### Past module 3.x serie
 
-The parameter `install_client` has been renamed to `manage_client` in version 4.0.0 of this module.
+ * The classes `snmp::server` and `snmp::trapd` has been merged into class `snmp`.  All of their class parameters available in the `snmp` class.
```
Comment (mmerfort) [inline README.md]: I think this should be “have been merged” because there are multiple classes. Also “the class”.

File: README.md [hunk 5]
```diff
@@ -59,13 +59,21 @@ include snmp
...
 
-The parameters `ro_community`, `rw_community`, `ro_network`, and `rw_network` will be removed in version 4.0.0 of this module.  The snmptrapd parameter name will become `authcommunity`.
+##### Current module 4.x serie
```
Comment (mmerfort) [inline README.md]: I think this should be “series”.

File: README.md (hunk 3 above)
Comment (Dan33l) [reply README.md]: version 3 is lower than version 4

Comment (alexjfisher) [reply README.md]: It was probably more the English didn't read too well.
Use `Puppet < 4`.  Problem solved! ;)

File: README.md (hunk 1 above)
Comment (alexjfisher) [reply README.md]: `serie` is also the French spelling I think.

File: README.md (hunk 3 above)
Comment (Dan33l) [reply README.md]: :D

**Pull request merged**